pom.xml

```diff
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>net.mekomsolutions</groupId>
+  <artifactId>openmrs-config-cambodia</artifactId>
+  <name>OpenMRS config for the JIB</name>
+  <description>OpenMRS configuration for the JIB</description>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <developers>
+    <developer>
+      <name>Mekom Solutions</name>
+    </developer>
+  </developers>
+  <organization>
+    <name>Mekom Solutions</name>
+    <url>http://www.mekomsolutions.com</url>
+  </organization>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>create-distribution</id>
```

Comment:
This should be something else. I realise the same string is there for Cambodia. We should move both to something like:
```xml

<id>create-package</id>
```

Reply:
yes, i used the Cambodian config as reference